core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -53,19 +51,15 @@ protected function createAuthProvider(ContainerBuilder $container, $id, $config,
     {
         $twoFactorProviderId = TwoFactorFactory::PROVIDER_ID_PREFIX.$id;
         $twoFactorFirewallConfigId = 'contao.security.two_factor_firewall_config.'.$id;
-        $twoFactorFirewallConfig = new Definition(TwoFactorFirewallConfig::class, [$config, $id, null]);
```

Comment:
this removes `$config` and `$id` from the firewall config definition

Reply:
True, fixed in 91d5177f7.